xray/conventions.py

```diff
@@ -35,7 +35,9 @@ def mask_and_scale(array, fill_value=None, scale_factor=None, add_offset=None,
         Original array of values to wrap
     fill_value : number, optional
         All values equal to fill_value in the original array are replaced
-        by NaN.
+        by NaN.  If an array of multiple values is provided a warning will be
```

Comment:
can you copy this to the docstring for `open_dataset` in `xray/backends/api.py`?